### DISCUSSION TITLE
Does wezterm support input method

### DISCUSSION THREAD

**Discussion by chaoyi** [wezterm/wezterm#611] [Q&A] [answered]
I'd like to type Chinese into the terminal.

**Comment by wez** [comment, marked as answer]
On macOS, yes, but you need to enable an option: https://wezfurlong.org/wezterm/config/keys.html#macos-and-the-input-method-editor-ime

On Windows: yes

On X11/Wayland: no: https://github.com/wez/wezterm/issues/250
